skopt/tests/test_common.py

```diff
@@ -422,7 +422,7 @@ def test_early_stopping_delta_x(minimizer):
     res = minimizer(bench1,
                     callback=DeltaXStopper(0.1),
                     dimensions=[(-1., 1.)],
-                    x0=[[0.1], [-0.1], [0.9]],
+                    x0=[[-0.1], [0.9], [0.1]],
```

Comment:
Strange that this changed anything, since those are the same points, but in a different order. 

I am merging in spite of this, that's not very important.